test/unittests/execute_test.cpp

```diff
@@ -1,21 +1,37 @@
 #include "execute.hpp"
 #include "limits.hpp"
 #include "parser.hpp"
+#include <gmock/gmock.h>
 #include <gtest/gtest.h>
 #include <test/utils/asserts.hpp>
 #include <test/utils/hex.hpp>
 
 using namespace fizzy;
 
+MATCHER(Trapped, "")
```

Comment:
I expect that `""` is used in failure messages. Please check how it looks add descriptions.

Reply:
Yes, the manual said that. Will need to come up with something.

Reply:
```
/Users/alex/Projects/fizzy/test/unittests/execute_test.cpp:34: Failure
Value of: execute(parse(wasm), 0, {})
Expected: trapped
  Actual: 32-byte object <00-CA 7C-0F 01-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00> (of type fizzy::execution_result)
```

It seems matchers are just lowercased. However need to have some kind of formatter for `execution_result`.